pkg/state/state.go

```diff
@@ -41,6 +41,13 @@ CREATE UNIQUE INDEX IF NOT EXISTS only_first_migration_without_parent ON %[1]s.m
 -- History is linear
 CREATE UNIQUE INDEX IF NOT EXISTS history_is_linear ON %[1]s.migrations (schema, parent);
 
+-- Add a column to tell whether the row represents an auto-detected DDL capture or a pgroll migration
+ALTER TABLE %[1]s.migrations ADD COLUMN IF NOT EXISTS migration_type
```

Comment:
wondering if we should add it to the create table statement too, so we get a full representation in the `CREATE TABLE` statement + first time users don't go through 2 table changes

Reply:
We could add it there. I did it this way because then the column definition is only in one place and there is no chance of existing users ending up with slightly different definition than than new users.

I also thought of this as more of a 'migration' being applied to the table.